package.json

```diff
@@ -28,6 +28,7 @@
     "integration-tests:alpha": "node build/testing/sauce-labs/test-browsers.js alpha",
     "integration-tests:beta": "node build/testing/sauce-labs/test-browsers.js beta",
     "integration-tests:release": "node build/testing/sauce-labs/test-browsers.js release",
+    "prepare": "lerna run prepare",
```

Comment:
What is the benefit of aliasing this in the root package and how does it align with this specific PR?

Reply:
Noted in the description, but, this removes the warning message in the CLI.

Reply:
This all goes towards getting started documentation as I got back into the code with a fresh perspective.

Reply:
I'm not aware of the CLI error - what was it?

Reply:
Actually, this also runs the prepare statements automatically saving time.  Basically, a user only has to run `yarn` to both install and prepare. reducing keystrokes.  Nick thought this made sense as well.

Reply:
There wasn't an error.  It was a warning.  but, after looking again it was just informational.  but, in my search to fix I found this trick to get prepare to autorun.

Reply:
Thanks for the FYI- I forgot prepare runs on install which is great.